src/helpers.php

```diff
@@ -3,9 +3,9 @@
 
```

Comment:
Just a thought, do we want to namespace our functions?
In that case we should probably make that change before people get too far and create too many plugins ;)